### PR TITLE
Gough Vol 4, 14-16

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: history-of-the-quakers-v4/original/01-book-6-chapter-14.adoc [hunk 1]
```diff
@@ -48,13 +59,19 @@ to the glory of God, and the benefit of these nations.
 May the King of Kings make thy reign long and glorious,
 to which temporal blessing we shall pray for thy eternal happiness.
 
+[.signed-section-context-close]
```
Comment (maijapapaya) [inline history-of-the-quakers-v4/original/01-book-6-chapter-14.adoc:62]: Not sure if this is the best way for this.

File: history-of-the-quakers-v4/original/02-book-6-chapter-15.adoc [hunk 2]
```diff
@@ -949,39 +957,17 @@ application being made to the Queen and Counsel, it produced the following order
...
+[.signed-section-context-open]
+At the Court of Kensington, the 11th day of October, 1705,
 
-Mr. Vernon,
+[.old-style]
+=== Present
 
-Mr. Earle.
+[.blurb]
+=== His Royal Highness Prince George of Denmark, Lord Archbishop of Canterbury, Lord Keeper, Lord Treasurer, Lord President, Duke of Somerset, Duke of Ormond, Earl of Ranelagh, Mr. Boyle, Mr. Secretary Hedges, Mr. Secretary Harley, Lord Chief Justice Holt, Lord Chief Justice Trevor, Mr. Vernon, Mr. Earle.
```
Comment (maijapapaya) [inline history-of-the-quakers-v4/original/02-book-6-chapter-15.adoc:970]: So I wasn't sure what to do with all this, I did what I thought looked okay, but idk if you wanted the names still in a list.

File: history-of-the-quakers-v4/original/02-book-6-chapter-15.adoc [hunk 3]
```diff
@@ -1035,15 +1027,25 @@ that in the delightful fruition thereof, the Queen may enjoy many days,
...
 
+[.signed-section-signature]
 John Field,
 
+[.signed-section-signature]
 Joseph Wyeth.
 
+--
+
+[.embedded-content-document]
```
Comment (maijapapaya) [inline history-of-the-quakers-v4/original/02-book-6-chapter-15.adoc:1038]: This is where I start forgetting and doubting what I should do with these little quotes.....maybe it just should have been a quote block?? AHHH!!!!

File: history-of-the-quakers-v4/original/02-book-6-chapter-15.adoc [hunk 4]
```diff
@@ -1035,15 +1027,25 @@ that in the delightful fruition thereof, the Queen may enjoy many days,
...
+--
+
+[.old-style]
 === The Queen`'s Answer.
```
Comment (maijapapaya) [inline history-of-the-quakers-v4/original/02-book-6-chapter-15.adoc:1042]: After I left the last comment above, I saw this and remembered why I embedded it....but still not sure if it's the right call.

File: history-of-the-quakers-v4/original/01-book-6-chapter-14.adoc (hunk 1 above)
Comment (jaredh159) [reply history-of-the-quakers-v4/original/01-book-6-chapter-14.adoc:62]: i think it seems good.

File: history-of-the-quakers-v4/original/02-book-6-chapter-15.adoc (hunk 4 above)
Comment (jaredh159) [reply history-of-the-quakers-v4/original/02-book-6-chapter-15.adoc:1072]: I think this is fine, it's a tough call, doesn't fit perfectly into any category.  This looks good though, so let's go with it.

File: history-of-the-quakers-v4/original/02-book-6-chapter-15.adoc (hunk 2 above)
Comment (jaredh159) [reply history-of-the-quakers-v4/original/02-book-6-chapter-15.adoc]: yeah, this is super strange.  I formatted a list almost exactly like this somewhere else, so i'm going to modify this one to match that one, but it's so rare i think it's not even worth you bothering trying to remember it, so I'll take care of it. 👍

**Pull request merged**